.gitattributes

```diff
@@ -1,2 +1,7 @@
 preview/MsixCore/Tests/*.msix filter=lfs diff=lfs merge=lfs -text
 preview/MsixCore/Tests/*.appx filter=lfs diff=lfs merge=lfs -text
+
+# Mark third party code as vendored

```

Comment:
What about src/test/msixtest/inc/catch.hpp

Should we actually move that under lib or a new external directory (or choose one that linguist already knows) that always contains vendored code?